design-documents/add-scope-to-cms-pages.md

```diff
@@ -0,0 +1,94 @@
+
+## [HLD] MC-13907: Add StoreView Scope to CMS Pages
+
+HLD for [MC-13907](https://jira.corp.magento.com/browse/MC-13907): Allow CMS Page to Be Edited and Viewed in Admin Based On StoreView
+
+### Terms
+
+### Overview
+CMS Pages can currently be turned on/off for store views, but the content cannot be customized. Epic MC-13907 will provide support for customizing a Page at the store view level so that users can more easily manage localized content.
```

Comment:
Can you provide more information on this existing functionality that would need to be removed? My understanding is the current workaround is to create multiple pages/blocks with different file names.

Reply:
When you save the page there is option, save and duplicate. Duplication functionality was added in 2.3 to simplify content localization. Seems likes it will no longer be needed after proposed implementation.